src/plugins/ecs/__init__.py

```diff
@@ -345,4 +389,81 @@ def information():
 	jarvis ecs list clusters [in <region/account>]
 	jarvis ecs list services <cluster> [in <region/account>]
 	jarvis ecs describe|desc <cluster> [in <region/account>]
-	jarvis ecs describe|desc <service> <cluster> [in <region/account>]"""
+	jarvis ecs describe|desc <service> <cluster> [in <region/account>]
+	jarvis ecs list tasks[-><task_name_optional>] running <cluster> [in <region/account>]"""
+
+
+#list the tasks in cluster
+def get_task_list(instance=None, next_token=None, cluster=None, ecs=None):
+    ''' Get the running tasks '''
+    running_tasks = []
+    if instance:
```

Comment:
You can remove all the instance stuff since you will not be running this for a particular instance - you will always be running it for the whole cluster...